tests/lua/basic.sy

```diff
@@ -0,0 +1,5 @@
+a : int : `return 1`
```

Comment:
These tests make me wonder if the return should be implicit? That way you write a lua expression, which might feel more natural?